arlas-tests/src/test/java/io/arlas/server/ogc/wfs/WFSServiceIT.java

```diff
@@ -72,6 +73,98 @@ public void testNoHeaderFilter() throws Exception {
         );
     }
 
+    @Test
+    public void testGetFeatureWithAvailableColumn() throws Exception {
+        //an empty column filter is not considered
+        handleGetFeatureNoHeaderFilter(
+                get(Arrays.asList(
+                        new ImmutablePair<>("SERVICE", "WFS"),
+                        new ImmutablePair<>("VERSION", "2.0.0"),
+                        new ImmutablePair<>("COUNT", "1000"),
+                        new ImmutablePair<>("REQUEST", "GetFeature")),
+                        new Filter(),
+                        Optional.of("")));
+
+        handleGetFeatureNoHeaderFilter(
+                get(Arrays.asList(
+                        new ImmutablePair<>("SERVICE", "WFS"),
+                        new ImmutablePair<>("VERSION", "2.0.0"),
+                        new ImmutablePair<>("COUNT", "1000"),
+                        new ImmutablePair<>("REQUEST", "GetFeature")),
+                        new Filter(),
+                        Optional.of("params.job,params.country,params.city")));
+    }
+
+    @Test
+    public void testGetFeatureWithUnavailableColumn() throws Exception {
```

Comment:
I would say : `WithNotReturnedColumn` instead of `WithUnavailableColumn`
Because at first, I thought it's a case where the column is unavailable and we throw an exception; which is not the case for this test